mevolib/cluster/Genes.py

```diff
@@ -125,215 +85,171 @@
-            Reference sequence of the same type as 'record'.
-        alignment_bin  ( string )
-            Binary path to the alignment tool that will be used in the
-            normalization process.
+    Args:
+        record: Sequence to normalize.
+        refseq_record: Reference sequence of the same type as 'record'.
+        alignment_bin: Binary path to the alignment tool that will be used in the normalization process.
 
-    Returns :
-        Bio.Seq
-            Normalized sequence of 'record'.
-        list
-            List of SeqFeature objects (from Biopython) from reference sequence.
+    Returns:
+        Bio.Seq: Normalized sequence of 'record'.
+        list: List of SeqFeature objects (from Biopython) from reference sequence.
 
-    Raises :
-        RuntimeError
-            If the call to the alignment tool command raises an exception.
+    Raises:
+        RuntimeError: If the call to the alignment tool command raises an exception.
     """
     tmpfile = tempfile.NamedTemporaryFile()
     SeqIO.write([refseq_record, record], tmpfile.name, 'fasta')
     alignment = align.get_alignment(alignment_bin, tmpfile.name, 'fasta')
-    # Get the normalized sequence by removing the sites that correspond to gaps
-    # introduced in the reference sequence during the alignment process
+    # Get the normalized sequence by removing the sites that correspond to gaps introduced in the reference 
+    # sequence during the alignment process
     record_seq = ''.join((x  for i, x in enumerate(alignment[1])
                                  if alignment[0][i] != '-'))
-    return ( Seq(record_seq, refseq_record.seq.alphabet),
-             refseq_record.features )
+    return (Seq(record_seq, refseq_record.seq.alphabet),
+             refseq_record.features)
 
 
-
-def _string_filter ( input_str ) :
-    """
-    Arguments :
-        input_str  ( string )
-            Input string to filter.
-
-    Returns :
-        string
-            Lower case of 'input_str' with known GenBank's misspellings
-            corrected.
+def _string_filter(input_str: str) -> str:
+    """Returns a string lower case of 'input_str' with known GenBank's misspellings corrected.
+    Args:
+        input_str: Input string to filter.
     """
     # Ignore case
     final_str = input_str.lower()
     # Correct misspellings
     final_str = final_str.replace('oxydase', 'oxidase')
-    return ( final_str )
-
-
-#-------------------------------------------------------------------------------
+    return final_str
 
-def get_features ( ) :
-    """
-    Returns :
-        list
-            List of all possible feature keywords that can be found in any
-            GenBank's sequence record.
-    """
-    return ( [x  for x in iter(viewkeys(_FEAT_QUAL_DICT))] )
 
+def get_features() -> List:
+    """Returns a list of all possible feature keywords that can be found in any GenBank's sequence record."""
+    return [x for x in iter(_FEAT_QUAL_DICT.keys())]
```

Comment:
Code improvement (has nothing to do with your update):
```suggestion
    return list(_FEAT_QUAL_DICT.keys())
```